osh/hub/osh_xmlrpc/scan.py

```diff
@@ -237,30 +237,36 @@ def find_tasks(request, query):
      * 'nvr': search by specific NVR
      * 'regex': find by provided regex (this is not match, but find, if you
                 want match, change your regex to "^<regex>$")
+     * 'comment': string, search by comment
+
+    Query also supports following optional keys:
+     * 'states': list, search by task states
 
     Returned list is ordered by date, when task finished -- latest task is
     first. Unfinished tasks are at the tail. If there is any problem with
     querying, empty list is returned.
     """
     if not isinstance(query, dict):
         return []
-    nvr = query.get('nvr')
     package_name = query.get('package_name')
-    comment = query.get('comment')
+    nvr = query.get('nvr')
     regex = query.get('regex')
+    comment = query.get('comment')
     states = query.get('states')
     latest = query.get('latest')
 
     result = []
-    tasks = None
```

Comment:
nit: You can use `tasks = Task.objects.none()` here directly.  Then, the newly added `else` branch on L267 is redundant.  This will also make sure that the `tasks` variable will be always set to some instance of `QuerySet`.

Reply:
updated.